opencti-platform/opencti-front/tests_e2e/filters/addFilter.spec.ts

```diff
@@ -1,12 +1,47 @@
 import { expect, test } from '../fixtures/baseFixtures';
 import FiltersUtils from '../model/filters.pageModel';
+import ReportPage from '../model/report.pageModel';
 
 test('Add a new filter in the observables list and check the filter is still present when we come back to the page', async ({ page }) => {
   await page.goto('/dashboard/observations/observables');
   const filterUtils = new FiltersUtils(page);
-  await filterUtils.addFilter('Entity type', 'Artifact');
+  await filterUtils.addFilterFromList('Entity type', 'Artifact');
   await expect(page.getByRole('button', { name: 'Entity type = Artifact' })).toBeVisible();
   await page.goto('/dashboard/');
   await page.goto('/dashboard/observations/observables');
   await expect(page.getByRole('button', { name: 'Entity type = Artifact' })).toBeVisible();
 });
+
+test('Add different filters with different operators', async ({ page }) => {
+  await page.goto('/dashboard/observations/observables');
```

Comment:
having multiple `test(...)` is good for readibility and all, but I think it makes the test suite very long to run.

Each call to `page.goto` takes forever to load, then everything is quite rapid.

If you could run all the tests that occur in the same page in the same `test()` that would make the suite run faster.
That's why the Reports CRUD tests done by @lndrtrbn & co. are all in one place.

Reply:
Indeed 👍